### PR TITLE
Eahw 2824/use kafka common GitHub

### PR DESCRIPTION
**Pull request by JWeston-HO**
This PR introduces:
- The ability to pull in github packages

### CODE REVIEW

File: .github/workflows/deploy.yaml
Comment (taomoh) [inline .github/workflows/deploy.yaml:1]: Why was a github workflow added? I thought we were keeping the builds in drone?

File: pom.xml [hunk 1]
```diff
@@ -49,15 +51,20 @@
             <artifactId>jakarta.validation-api</artifactId>
             <version>3.0.2</version>
         </dependency>
+        <dependency>
+            <groupId>jakarta.persistence</groupId>
+            <artifactId>jakarta.persistence-api</artifactId>
+            <version>3.1.0</version>
+        </dependency>
```
Comment (taomoh) [inline pom.xml:58]: I don't see why this dependency started to be necessary

File: .github/workflows/deploy.yaml
Comment (JWeston-HO) [reply .github/workflows/deploy.yaml:1]: We are keeping builds in Drone, That was left over from the testing we did using github actions. It has been removed.

File: pom.xml (hunk 1 above)
Comment (JWeston-HO) [reply pom.xml]: I am not sure. I must have added it to the wrong repo by mistake. Removed now

**Review by taomoh — APPROVED** [review]

**Review by sewerynmi — APPROVED** [review]

**Pull request merged**